bindings/wasm/src/credential/presentation.rs

```diff
@@ -8,7 +8,9 @@ use identity::credential::Presentation;
 use identity::credential::PresentationBuilder;
```

Comment:
Update the licence header to 2022 in this file (bindings/wasm/src/credential/presentation.rs).